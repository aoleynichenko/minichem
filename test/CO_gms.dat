 $DATA  
CO molecule                                                                     
C1       0
CARBON      6.0      0.0000000000      0.0000000000      0.0000000000
   STO     3
        
OXYGEN      8.0      0.0000000000      0.0000000000      1.1454782408
   STO     3
        
 $END      
--- CLOSED SHELL ORBITALS --- GENERATED AT Tue Jan  1 22:19:58 2019
CO molecule                                                                     
E(RHF)=     -111.2254495171, E(NUC)=   22.1745879222,   10 ITERS
 $VEC   
 1  1 4.23172499E-04-8.30735239E-03 0.00000000E+00 0.00000000E+00-7.14507347E-03
 1  2 9.94184127E-01 2.73366403E-02 0.00000000E+00 0.00000000E+00-6.55115819E-03
 2  1 9.93626299E-01 2.62034262E-02-0.00000000E+00-0.00000000E+00 6.84875484E-03
 2  2-1.29309587E-04-6.84906030E-03-0.00000000E+00-0.00000000E+00 1.20960654E-03
 3  1-1.23840090E-01 2.43672549E-01 0.00000000E+00 0.00000000E+00 1.65889547E-01
 3  2-2.22538196E-01 7.70576042E-01 0.00000000E+00 0.00000000E+00-2.10490523E-01
 4  1 1.69575746E-01-5.58937061E-01 0.00000000E+00 0.00000000E+00-6.48721919E-02
 4  2-1.31698796E-01 6.42520390E-01 0.00000000E+00 0.00000000E+00 6.14714460E-01
 5  1-0.00000000E+00-0.00000000E+00-1.78343199E-01 4.08385203E-01-0.00000000E+00
 5  2-0.00000000E+00-0.00000000E+00-3.17838725E-01 7.27813748E-01-0.00000000E+00
 6  1-0.00000000E+00-0.00000000E+00 4.08385203E-01 1.78343199E-01-0.00000000E+00
 6  2-0.00000000E+00-0.00000000E+00 7.27813748E-01 3.17838725E-01-0.00000000E+00
 7  1-1.65078345E-01 7.47715507E-01 0.00000000E+00 0.00000000E+00-5.74717060E-01
 7  2-1.41217248E-03 4.91451590E-02 0.00000000E+00 0.00000000E+00 4.44578797E-01
 8  1 0.00000000E+00 0.00000000E+00 8.89988646E-01-2.66591035E-01 0.00000000E+00
 8  2 0.00000000E+00 0.00000000E+00-6.28898185E-01 1.88382873E-01 0.00000000E+00
 9  1 0.00000000E+00 0.00000000E+00 2.66591035E-01 8.89988646E-01 0.00000000E+00
 9  2 0.00000000E+00 0.00000000E+00-1.88382873E-01-6.28898185E-01 0.00000000E+00
10  1-1.22427418E-01 9.37401236E-01 0.00000000E+00 0.00000000E+00 1.20585470E+00
10  2 1.26423390E-01-1.04106493E+00 0.00000000E+00 0.00000000E+00 9.58468125E-01
 $END   
 POPULATION ANALYSIS
CARBON       5.80106   0.19894   5.95324   0.04676
OXYGEN       8.19894  -0.19894   8.04676  -0.04676
 MOMENTS AT POINT    1 X,Y,Z=  0.000000  0.000000  1.236768
 DIPOLE       0.000000  0.000000  0.124381
 QUADRUPOLE   1.508014  1.508014 -3.016029  0.000000  0.000000  0.000000
 2ND MOMENTS -8.704310 -8.704310-11.720339  0.000000  0.000000  0.000000
