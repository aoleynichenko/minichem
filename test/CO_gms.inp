 $CONTRL SCFTYP=RHF RUNTYP=ENERGY UNITS=BOHR $END
 $SYSTEM TIMLIM=1 $END
 $STATPT OPTTOL=1.0E-5  $END
 $BASIS  GBASIS=STO NGAUSS=3 $END
 $GUESS  GUESS=HUCKEL $END
 $ELMOM IEMOM=2 $END
 $DATA
CO molecule
C1
CARBON   6.0   0.0   0.0   0.0
OXYGEN   8.0   0.0   0.0   2.16464
 $END
