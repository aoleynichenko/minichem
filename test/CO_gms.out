----- GAMESS execution script 'rungms' -----
This job is running on host MBP-Aleksandr-2
under operating system Darwin at вторник, 1 января 2019 г. 22:19:58 (MSK)
Available scratch disk space (Kbyte units) at beginning of the job is
Filesystem   1024-blocks      Used Available Capacity  iused   ifree %iused  Mounted on
/dev/disk0s2   155273440 130904084  24113356    85% 32790019 6028339   84%   /
GAMESS temporary binary files will be written to ./
GAMESS supplementary output files will be written to ./
Copying input file CO_gms.inp to your run's scratch directory...

 Distributed Data Interface kickoff program.
 Initiating 1 compute processes on 1 nodes to run the following command:
 /usr/local/gamess/gamess.00.x CO_gms 

          ******************************************************
          *         GAMESS VERSION = 18 AUG 2016 (R1)          *
          *             FROM IOWA STATE UNIVERSITY             *
          * M.W.SCHMIDT, K.K.BALDRIDGE, J.A.BOATZ, S.T.ELBERT, *
          *   M.S.GORDON, J.H.JENSEN, S.KOSEKI, N.MATSUNAGA,   *
          *          K.A.NGUYEN, S.J.SU, T.L.WINDUS,           *
          *       TOGETHER WITH M.DUPUIS, J.A.MONTGOMERY       *
          *         J.COMPUT.CHEM.  14, 1347-1363(1993)        *
          **************** 64 BIT APPLE VERSION ****************

  SINCE 1993, STUDENTS AND POSTDOCS WORKING AT IOWA STATE UNIVERSITY
  AND ALSO IN THEIR VARIOUS JOBS AFTER LEAVING ISU HAVE MADE IMPORTANT
  CONTRIBUTIONS TO THE CODE:
     IVANA ADAMOVIC, CHRISTINE AIKENS, YURI ALEXEEV, POOJA ARORA,
     ANDREY ASADCHEV, ROB BELL, PRADIPTA BANDYOPADHYAY, JONATHAN BENTZ,
     BRETT BODE, KURT BRORSEN, CALEB CARLIN, GALINA CHABAN, WEI CHEN,
     CHEOL HO CHOI, PAUL DAY, ALBERT DEFUSCO, NUWAN DESILVA, TIM DUDLEY,
     DMITRI FEDOROV, GRAHAM FLETCHER, MARK FREITAG, KURT GLAESEMANN,
     DAN KEMP, GRANT MERRILL, NORIYUKI MINEZAWA, JONATHAN MULLIN,
     TAKESHI NAGATA, SEAN NEDD, HEATHER NETZLOFF, BOSILJKA NJEGIC, RYAN OLSON,
     MIKE PAK, SPENCER PRUITT, LUKE ROSKOP, JIM SHOEMAKER, LYUDMILA SLIPCHENKO,
     TONY SMITH, SAROM SOK LEANG, JIE SONG, TETSUYA TAKETSUGU, SIMON WEBB,
     PENG XU, SOOHAENG YOO, FEDERICO ZAHARIEV

  ADDITIONAL CODE HAS BEEN PROVIDED BY COLLABORATORS IN OTHER GROUPS:
     IOWA STATE UNIVERSITY:
          JOE IVANIC, AARON WEST, LAIMUTIS BYTAUTAS, KLAUS RUEDENBERG
     UNIVERSITY OF TOKYO: KIMIHIKO HIRAO, TAKAHITO NAKAJIMA,
          TAKAO TSUNEDA, MUNEAKI KAMIYA, SUSUMU YANAGISAWA,
          KIYOSHI YAGI, MAHITO CHIBA, SEIKEN TOKURA, NAOAKI KAWAKAMI
     UNIVERSITY OF AARHUS: FRANK JENSEN
     UNIVERSITY OF IOWA: VISVALDAS KAIRYS, HUI LI
     NATIONAL INST. OF STANDARDS AND TECHNOLOGY: WALT STEVENS, DAVID GARMER
     UNIVERSITY OF PISA: BENEDETTA MENNUCCI, JACOPO TOMASI
     UNIVERSITY OF MEMPHIS: HENRY KURTZ, PRAKASHAN KORAMBATH
     UNIVERSITY OF ALBERTA: TOBY ZENG, MARIUSZ KLOBUKOWSKI
     UNIVERSITY OF NEW ENGLAND: MARK SPACKMAN
     MIE UNIVERSITY: HIROAKI UMEDA
     NAT. INST. OF ADVANCED INDUSTRIAL SCIENCE AND TECHNOLOGY: KAZUO KITAURA
     MICHIGAN STATE UNIVERSITY:
          KAROL KOWALSKI, MARTA WLOCH, JEFFREY GOUR, JESSE LUTZ,
          WEI LI, PIOTR PIECUCH
     UNIVERSITY OF SILESIA: MONIKA MUSIAL, STANISLAW KUCHARSKI
     FACULTES UNIVERSITAIRES NOTRE-DAME DE LA PAIX:
          OLIVIER QUINET, BENOIT CHAMPAGNE
     UNIVERSITY OF CALIFORNIA - SANTA BARBARA: BERNARD KIRTMAN
     INSTITUTE FOR MOLECULAR SCIENCE:
          KAZUYA ISHIMURA, MICHIO KATOUDA, AND SHIGERU NAGASE
     UNIVERSITY OF NOTRE DAME: ANNA POMOGAEVA, DAN CHIPMAN
     KYUSHU UNIVERSITY:
          HARUYUKI NAKANO,
          FENG LONG GU, JACEK KORCHOWIEC, MARCIN MAKOWSKI, AND YURIKO AOKI,
          HIROTOSHI MORI AND EISAKU MIYOSHI
     PENNSYLVANIA STATE UNIVERSITY:
          TZVETELIN IORDANOV, CHET SWALINA, JONATHAN SKONE,
          SHARON HAMMES-SCHIFFER
     WASEDA UNIVERSITY:
          MASATO KOBAYASHI, TOMOKO AKAMA, TSUGUKI TOUMA,
          TAKESHI YOSHIKAWA, YASUHIRO IKABATA, JUNJI SEINO,
          YUYA NAKAJIMA, HIROMI NAKAI
     NANJING UNIVERSITY: SHUHUA LI
     UNIVERSITY OF NEBRASKA:
          PEIFENG SU, DEJUN SI, NANDUN THELLAMUREGE, YALI WANG, HUI LI
     UNIVERSITY OF ZURICH: ROBERTO PEVERATI, KIM BALDRIDGE
     N. COPERNICUS UNIVERSITY AND JACKSON STATE UNIVERSITY:
          MARIA BARYSZ
     UNIVERSITY OF COPENHAGEN: Jimmy Kromann, CASPER STEINMANN
     TOKYO INSTITUTE OF TECHNOLOGY: HIROYA NAKATA
     NAGOYA UNIVERSITY: YOSHIO NISHIMOTO, STEPHAN IRLE
     MOSCOW STATE UNIVERSITY: VLADIMIR MIRONOV

 EXECUTION OF GAMESS BEGUN Tue Jan  1 22:19:58 2019

            ECHO OF THE FIRST FEW INPUT CARDS -
 INPUT CARD> $CONTRL SCFTYP=RHF RUNTYP=ENERGY UNITS=BOHR $END                               
 INPUT CARD> $SYSTEM TIMLIM=1 $END                                                          
 INPUT CARD> $STATPT OPTTOL=1.0E-5  $END                                                    
 INPUT CARD> $BASIS  GBASIS=STO NGAUSS=3 $END                                               
 INPUT CARD> $GUESS  GUESS=HUCKEL $END                                                      
 INPUT CARD> $ELMOM IEMOM=2 $END                                                            
 INPUT CARD> $DATA                                                                          
 INPUT CARD>CO molecule                                                                     
 INPUT CARD>C1                                                                              
 INPUT CARD>CARBON   6.0   0.0   0.0   0.0                                                  
 INPUT CARD>OXYGEN   8.0   0.0   0.0   2.16464                                              
 INPUT CARD> $END                                                                           
    1000000 WORDS OF MEMORY AVAILABLE

     BASIS OPTIONS
     -------------
     GBASIS=STO          IGAUSS=       3      POLAR=NONE    
     NDFUNC=       0     NFFUNC=       0     DIFFSP=       F
     NPFUNC=       0      DIFFS=       F     BASNAM=        


     RUN TITLE
     ---------
 CO molecule                                                                     

 THE POINT GROUP OF THE MOLECULE IS C1      
 THE ORDER OF THE PRINCIPAL AXIS IS     0

 ATOM      ATOMIC                      COORDINATES (BOHR)
           CHARGE         X                   Y                   Z
 CARBON      6.0     0.0000000000        0.0000000000        0.0000000000
 OXYGEN      8.0     0.0000000000        0.0000000000        2.1646400000

          INTERNUCLEAR DISTANCES (ANGS.)
          ------------------------------

                1 CARB       2 OXYG  

   1 CARB    0.0000000    1.1454782 *
   2 OXYG    1.1454782 *  0.0000000  

  * ... LESS THAN  3.000


     ATOMIC BASIS SET
     ----------------
 THE CONTRACTED PRIMITIVE FUNCTIONS HAVE BEEN UNNORMALIZED
 THE CONTRACTED BASIS FUNCTIONS ARE NOW NORMALIZED TO UNITY

  SHELL TYPE  PRIMITIVE        EXPONENT          CONTRACTION COEFFICIENT(S)

 CARBON    

      1   S       1            71.6168373    0.154328967295
      1   S       2            13.0450963    0.535328142282
      1   S       3             3.5305122    0.444634542185

      2   L       4             2.9412494   -0.099967229187    0.155916274999
      2   L       5             0.6834831    0.399512826089    0.607683718598
      2   L       6             0.2222899    0.700115468880    0.391957393099

 OXYGEN    

      3   S       7           130.7093214    0.154328967295
      3   S       8            23.8088661    0.535328142282
      3   S       9             6.4436083    0.444634542185

      4   L      10             5.0331513   -0.099967229187    0.155916274999
      4   L      11             1.1695961    0.399512826089    0.607683718598
      4   L      12             0.3803890    0.700115468880    0.391957393099

 TOTAL NUMBER OF BASIS SET SHELLS             =    4
 NUMBER OF CARTESIAN GAUSSIAN BASIS FUNCTIONS =   10
 NUMBER OF ELECTRONS                          =   14
 CHARGE OF MOLECULE                           =    0
 SPIN MULTIPLICITY                            =    1
 NUMBER OF OCCUPIED ORBITALS (ALPHA)          =    7
 NUMBER OF OCCUPIED ORBITALS (BETA )          =    7
 TOTAL NUMBER OF ATOMS                        =    2
 THE NUCLEAR REPULSION ENERGY IS       22.1745879222

 THIS MOLECULE IS RECOGNIZED AS BEING LINEAR,
 ORBITAL LZ DEGENERACY TOLERANCE ETOLLZ= 1.00E-06

     $CONTRL OPTIONS
     ---------------
 SCFTYP=RHF          RUNTYP=ENERGY       EXETYP=RUN     
 MPLEVL=       0     CITYP =NONE         CCTYP =NONE         VBTYP =NONE    
 DFTTYP=NONE         TDDFT =NONE    
 MULT  =       1     ICHARG=       0     NZVAR =       0     COORD =UNIQUE  
 PP    =NONE         RELWFN=NONE         LOCAL =NONE         NUMGRD=       F
 ISPHER=      -1     NOSYM =       0     MAXIT =      30     UNITS =BOHR    
 PLTORB=       F     MOLPLT=       F     AIMPAC=       F     FRIEND=        
 NPRINT=       7     IREST =       0     GEOM  =INPUT   
 NORMF =       0     NORMP =       0     ITOL  =      20     ICUT  =       9
 INTTYP=BEST         GRDTYP=BEST         QMTTOL= 1.0E-06

     $SYSTEM OPTIONS
     ---------------
  REPLICATED MEMORY=     1000000 WORDS (ON EVERY NODE).
 DISTRIBUTED MEMDDI=           0 MILLION WORDS IN AGGREGATE,
 MEMDDI DISTRIBUTED OVER   1 PROCESSORS IS           0 WORDS/PROCESSOR.
 TOTAL MEMORY REQUESTED ON EACH PROCESSOR=     1000000 WORDS.
 TIMLIM=           1.00 MINUTES, OR       0.0 DAYS.
 PARALL= F  BALTYP=  DLB     KDIAG=    0  COREFL= F
 MXSEQ2=     300 MXSEQ3=     150  mem10=         0

          ----------------
          PROPERTIES INPUT
          ----------------

     MOMENTS            FIELD           POTENTIAL          DENSITY
 IEMOM =       2   IEFLD =       0   IEPOT =       0   IEDEN =       0
 WHERE =COMASS     WHERE =NUCLEI     WHERE =NUCLEI     WHERE =NUCLEI  
 OUTPUT=BOTH       OUTPUT=BOTH       OUTPUT=BOTH       OUTPUT=BOTH    
 IEMINT=       0   IEFINT=       0                     IEDINT=       0
                                                       MORB  =       0
          EXTRAPOLATION IN EFFECT
          SOSCF IN EFFECT
 ORBITAL PRINTING OPTION: NPREO=     1    10     2     1

     -------------------------------
     INTEGRAL TRANSFORMATION OPTIONS
     -------------------------------
     NWORD  =            0
     CUTOFF = 1.0E-09     MPTRAN =       0
     DIRTRF =       F     AOINTS =DUP     

          ----------------------
          INTEGRAL INPUT OPTIONS
          ----------------------
 NOPK  =       1 NORDER=       0 SCHWRZ=       F

     ------------------------------------------
     THE POINT GROUP IS C1 , NAXIS= 0, ORDER= 1
     ------------------------------------------

     DIMENSIONS OF THE SYMMETRY SUBSPACES ARE
 A   =   10

 ..... DONE SETTING UP THE RUN .....
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS   0.00%

          ********************
          1 ELECTRON INTEGRALS
          ********************
 ...... END OF ONE-ELECTRON INTEGRALS ......
 STEP CPU TIME =     0.02 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS 100.00%

          -------------
          GUESS OPTIONS
          -------------
          GUESS =HUCKEL            NORB  =       0          NORDER=       0
          MIX   =       F          PRTMO =       F          PUNMO =       F
          TOLZ  = 1.0E-08          TOLE  = 1.0E-05
          SYMDEN=       F          PURIFY=       F

 INITIAL GUESS ORBITALS GENERATED BY HUCKEL   ROUTINE.
 HUCKEL GUESS REQUIRES      2626 WORDS.

 SYMMETRIES FOR INITIAL GUESS ORBITALS FOLLOW.   BOTH SET(S).
     7 ORBITALS ARE OCCUPIED (    2 CORE ORBITALS).
     3=A        4=A        5=A        6=A        7=A        8=A        9=A   
    10=A   
 ...... END OF INITIAL ORBITAL SELECTION ......
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS 100.00%

                    ----------------------
                    AO INTEGRAL TECHNOLOGY
                    ----------------------
     S,P,L SHELL ROTATED AXIS INTEGRALS, REPROGRAMMED BY
        KAZUYA ISHIMURA (IMS) AND JOSE SIERRA (SYNSTAR).
     S,P,D,L SHELL ROTATED AXIS INTEGRALS PROGRAMMED BY
        KAZUYA ISHIMURA (INSTITUTE FOR MOLECULAR SCIENCE).
     S,P,D,F,G SHELL TO TOTAL QUARTET ANGULAR MOMENTUM SUM 5,
        ERIC PROGRAM BY GRAHAM FLETCHER (ELORET AND NASA ADVANCED
        SUPERCOMPUTING DIVISION, AMES RESEARCH CENTER).
     S,P,D,F,G,L SHELL GENERAL RYS QUADRATURE PROGRAMMED BY
        MICHEL DUPUIS (PACIFIC NORTHWEST NATIONAL LABORATORY).

          --------------------
          2 ELECTRON INTEGRALS
          --------------------

 THE -PK- OPTION IS OFF, THE INTEGRALS ARE NOT IN SUPERMATRIX FORM.
 STORING   15000 INTEGRALS/RECORD ON DISK, USING 12 BYTES/INTEGRAL.
 TWO ELECTRON INTEGRAL EVALUATION REQUIRES   89366 WORDS OF MEMORY.
 II,JST,KST,LST =  1  1  1  1 NREC =         1 INTLOC =    1
 II,JST,KST,LST =  2  1  1  1 NREC =         1 INTLOC =    2
 II,JST,KST,LST =  3  1  1  1 NREC =         1 INTLOC =   34
 II,JST,KST,LST =  4  1  1  1 NREC =         1 INTLOC =   83
 TOTAL NUMBER OF NONZERO TWO-ELECTRON INTEGRALS =                 511
          1 INTEGRAL RECORDS WERE STORED ON DISK FILE  8.
  ...... END OF TWO-ELECTRON INTEGRALS .....
 STEP CPU TIME =     0.01 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS  75.00%

          --------------------------
                 RHF SCF CALCULATION
          --------------------------

     NUCLEAR ENERGY =        22.1745879222
     MAXIT =   30     NPUNCH=    2
     EXTRAP=T  DAMP=F  SHIFT=F  RSTRCT=F  DIIS=F  DEM=F  SOSCF=T
     DENSITY MATRIX CONV=  1.00E-05
     SOSCF WILL OPTIMIZE      21 ORBITAL ROTATIONS, SOGTOL=   0.250
     MEMORY REQUIRED FOR RHF ITERS=     30924 WORDS.

 ITER EX DEM     TOTAL ENERGY        E CHANGE  DENSITY CHANGE     ORB. GRAD
   1  0  0     -110.9936663313  -110.9936663313   0.570762246   0.000000000
          ---------------START SECOND ORDER SCF---------------
   2  1  0     -111.1724854218    -0.1788190905   0.234731547   0.124060006
   3  2  0     -111.2130369921    -0.0405515703   0.079832640   0.059413491
   4  3  0     -111.2252821759    -0.0122451838   0.020097105   0.008926435
   5  4  0     -111.2254428266    -0.0001606506   0.005553106   0.001753343
   6  5  0     -111.2254493679    -0.0000065413   0.000384485   0.000129371
   7  6  0     -111.2254494894    -0.0000001215   0.000273824   0.000054283
   8  7  0     -111.2254495169    -0.0000000275   0.000013843   0.000013739
   9  8  0     -111.2254495171    -0.0000000002   0.000004143   0.000002275
  10  9  0     -111.2254495171    -0.0000000000   0.000000129   0.000000042

          -----------------
          DENSITY CONVERGED
          -----------------
     TIME TO FORM FOCK OPERATORS=       0.0 SECONDS (       0.0 SEC/ITER)
     TIME TO SOLVE SCF EQUATIONS=       0.0 SECONDS (       0.0 SEC/ITER)

 FINAL RHF ENERGY IS     -111.2254495171 AFTER  10 ITERATIONS

 LZ VALUE ANALYSIS FOR THE MOS
 ----------------------------------------
 MO     1 (    1) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     2 (    2) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     3 (    3) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     4 (    4) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     5 (    5) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO     6 (    5) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO     7 (    6) HAS LZ(WEIGHT)= 0.00(100.0%) 
 MO     8 (    7) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO     9 (    7) HAS LZ(WEIGHT)=-1.00( 50.0%)  1.00( 50.0%) 
 MO    10 (    8) HAS LZ(WEIGHT)= 0.00(100.0%) 

          ------------
          EIGENVECTORS
          ------------

                      1          2          3          4          5
                  -20.4156   -11.0922    -1.4453    -0.6968    -0.5399
                     A          A          A          A          A   
    1  C  1  S    0.000423   0.993626  -0.123840   0.169576  -0.000000
    2  C  1  S   -0.008307   0.026203   0.243673  -0.558937  -0.000000
    3  C  1  X    0.000000  -0.000000   0.000000   0.000000  -0.178343
    4  C  1  Y    0.000000  -0.000000   0.000000   0.000000   0.408385
    5  C  1  Z   -0.007145   0.006849   0.165890  -0.064872  -0.000000
    6  O  2  S    0.994184  -0.000129  -0.222538  -0.131699  -0.000000
    7  O  2  S    0.027337  -0.006849   0.770576   0.642520  -0.000000
    8  O  2  X    0.000000  -0.000000   0.000000   0.000000  -0.317839
    9  O  2  Y    0.000000  -0.000000   0.000000   0.000000   0.727814
   10  O  2  Z   -0.006551   0.001210  -0.210491   0.614714  -0.000000

                      6          7          8          9         10
                   -0.5399    -0.4451     0.3061     0.3061     1.0091
                     A          A          A          A          A   
    1  C  1  S   -0.000000  -0.165078   0.000000   0.000000  -0.122427
    2  C  1  S   -0.000000   0.747716   0.000000   0.000000   0.937401
    3  C  1  X    0.408385   0.000000   0.889989   0.266591   0.000000
    4  C  1  Y    0.178343   0.000000  -0.266591   0.889989   0.000000
    5  C  1  Z   -0.000000  -0.574717   0.000000   0.000000   1.205855
    6  O  2  S   -0.000000  -0.001412   0.000000   0.000000   0.126423
    7  O  2  S   -0.000000   0.049145   0.000000   0.000000  -1.041065
    8  O  2  X    0.727814   0.000000  -0.628898  -0.188383   0.000000
    9  O  2  Y    0.317839   0.000000   0.188383  -0.628898   0.000000
   10  O  2  Z   -0.000000   0.444579   0.000000   0.000000   0.958468
 ...... END OF RHF CALCULATION ......
 STEP CPU TIME =     0.01 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.0 SECONDS, CPU UTILIZATION IS 100.00%

     ----------------------------------------------------------------
     PROPERTY VALUES FOR THE RHF   SELF-CONSISTENT FIELD WAVEFUNCTION
     ----------------------------------------------------------------

          -----------------
          ENERGY COMPONENTS
          -----------------

         WAVEFUNCTION NORMALIZATION =       1.0000000000

                ONE ELECTRON ENERGY =    -196.4503863056
                TWO ELECTRON ENERGY =      63.0503488662
           NUCLEAR REPULSION ENERGY =      22.1745879222
                                      ------------------
                       TOTAL ENERGY =    -111.2254495171

 ELECTRON-ELECTRON POTENTIAL ENERGY =      63.0503488662
  NUCLEUS-ELECTRON POTENTIAL ENERGY =    -306.8495462053
   NUCLEUS-NUCLEUS POTENTIAL ENERGY =      22.1745879222
                                      ------------------
             TOTAL POTENTIAL ENERGY =    -221.6246094168
               TOTAL KINETIC ENERGY =     110.3991598997
                 VIRIAL RATIO (V/T) =       2.0074845644

  ...... PI ENERGY ANALYSIS ......

 ENERGY ANALYSIS:
            FOCK ENERGY=    -70.3496885837
          BARE H ENERGY=   -196.4503863056
     ELECTRONIC ENERGY =   -133.4000374446
         KINETIC ENERGY=    110.3991598997
          N-N REPULSION=     22.1745879222
           TOTAL ENERGY=   -111.2254495224
        SIGMA PART(1+2)=   -115.2592366045
               (K,V1,2)=    102.4811031753   -264.8095936653     47.0692538854
           PI PART(1+2)=    -18.1408008401
               (K,V1,2)=      7.9180567244    -42.0399525400     15.9810949755
  SIGMA SKELETON, ERROR=    -93.0846486823      0.0000000000
             MIXED PART= 0.00000E+00 0.00000E+00 0.00000E+00 0.00000E+00
 ...... END OF PI ENERGY ANALYSIS ......

          ---------------------------------------
          MULLIKEN AND LOWDIN POPULATION ANALYSES
          ---------------------------------------

     ATOMIC MULLIKEN POPULATION IN EACH MOLECULAR ORBITAL

                      1          2          3          4          5

                  2.000000   2.000000   2.000000   2.000000   2.000000

    1            -0.001940   2.000911   0.480807   0.530303   0.567850
    2             2.001940  -0.000911   1.519193   1.469697   1.432150

                      6          7

                  2.000000   2.000000

    1             0.567850   1.655283
    2             1.432150   0.344717

               ----- POPULATIONS IN EACH AO -----
                             MULLIKEN      LOWDIN
              1  C  1  S      1.99801     1.99659
              2  C  1  S      1.68788     1.67433
              3  C  1  X      0.56785     0.58060
              4  C  1  Y      0.56785     0.58060
              5  C  1  Z      0.97947     1.12111
              6  O  2  S      1.99795     1.99737
              7  O  2  S      1.85735     1.70258
              8  O  2  X      1.43215     1.41940
              9  O  2  Y      1.43215     1.41940
             10  O  2  Z      1.47933     1.50801

          ----- MULLIKEN ATOMIC OVERLAP POPULATIONS -----
          (OFF-DIAGONAL ELEMENTS NEED TO BE MULTIPLIED BY 2)

             1           2

    1    5.2782043
    2    0.5228601   7.6760755

          TOTAL MULLIKEN AND LOWDIN ATOMIC POPULATIONS
       ATOM         MULL.POP.    CHARGE          LOW.POP.     CHARGE
    1 CARBON        5.801064    0.198936         5.953238    0.046762
    2 OXYGEN        8.198936   -0.198936         8.046762   -0.046762

          -------------------------------
          BOND ORDER AND VALENCE ANALYSIS     BOND ORDER THRESHOLD=0.050
          -------------------------------

                   BOND                       BOND                       BOND
  ATOM PAIR DIST  ORDER      ATOM PAIR DIST  ORDER      ATOM PAIR DIST  ORDER
    1   2  1.145  2.515

                       TOTAL       BONDED        FREE
      ATOM            VALENCE     VALENCE     VALENCE
    1 CARBON            2.515       2.515       0.000
    2 OXYGEN            2.515       2.515       0.000

          ---------------------
          ELECTROSTATIC MOMENTS
          ---------------------

 POINT   1           X           Y           Z (BOHR)    CHARGE
                 0.000000    0.000000    1.236768        0.00 (A.U.)
         DX          DY          DZ         /D/  (DEBYE)
     0.000000    0.000000    0.124381    0.124381
        QXX         QYY         QZZ         QXY         QXZ         QYZ  (BUCKINGHAMS)
     1.508014    1.508014   -3.016029    0.000000    0.000000    0.000000
 ...... END OF PROPERTY EVALUATION ......
 STEP CPU TIME =     0.00 TOTAL CPU TIME =        0.0 (    0.0 MIN)
 TOTAL WALL CLOCK TIME=        0.1 SECONDS, CPU UTILIZATION IS  80.00%
               580000  WORDS OF DYNAMIC MEMORY USED
 EXECUTION OF GAMESS TERMINATED NORMALLY Tue Jan  1 22:19:58 2019
 DDI: 263640 bytes (0.3 MB / 0 MWords) used by master data server.

 ----------------------------------------
 CPU timing information for all processes
 ========================================
 0: 0.34 + 0.16 = 0.51
 1: 0.02 + 0.03 = 0.05
 ----------------------------------------
 ddikick.x: exited gracefully.
----- accounting info -----
Files used on the master node MBP-Aleksandr-2 were:
-rw-r--r--@ 1 Oleynichenko  staff      278  1 янв 22:19 .//CO_gms.F05
-rw-r--r--  1 Oleynichenko  staff   180016  1 янв 22:19 .//CO_gms.F08
-rw-r--r--  1 Oleynichenko  staff  1734160  1 янв 22:20 .//CO_gms.F10
-rw-r--r--  1 Oleynichenko  staff     2521  1 янв 22:19 .//CO_gms.dat
-rw-r--r--@ 1 Oleynichenko  staff      278  1 янв 22:19 .//CO_gms.inp
-rw-r--r--  1 Oleynichenko  staff    23083  1 янв 22:20 .//CO_gms.out
вторник,  1 января 2019 г. 22:20:01 (MSK)
0.155u 0.110s 0:03.28 7.9%	0+0k 0+4io 5pf+0w
